go/lambda-api-ci/lib/lambdaapistack/lambdaapistack.go

```diff
@@ -0,0 +1,56 @@
+package lambdaapistack
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-cdk-go/awscdk"
+	"github.com/aws/aws-cdk-go/awscdk/awsapigateway"
+	"github.com/aws/aws-cdk-go/awscdk/awsiam"
+	"github.com/aws/aws-cdk-go/awscdk/awslambda"
+	"github.com/aws/aws-cdk-go/awscdk/awss3"
+	"github.com/aws/jsii-runtime-go"
+)
+
+type CDKExampleLambdaApiStack awscdk.Stack
+
+type CDKExampleLambdaApiStackProps struct {
+	awscdk.StackProps
+	FunctionName *string
+}
+
+func NewCDKExampleLambdaApiStack(scope awscdk.Construct, id *string, props *CDKExampleLambdaApiStackProps) CDKExampleLambdaApiStack {
+	stack := awscdk.NewStack(scope, id, &props.StackProps)
```

Comment:
What happens if `props` is `nil`?

Reply:
That's a good question - I don't know :)